### DISCUSSION TITLE
Teaching More Than One Subject to the Same Class

### DISCUSSION THREAD

**Discussion by mattjmoore** [bytedeck/bytedeck#1674] [Q&A]
I work in an alternate program.  I teach four subjects to the same group of students.  In Bytedeck I'd love to have a campaign of Alternate Program Orientation Quests to start the year, then have each of their four subjects tracked with a separate XP total.  Is the only way to do this to organize the orientation into its own course, then have each academic subject organized as its own course, or is there a way to house all of this content within the same "course," but have students accumulate separate XP totals for each subject within that one course?

**Comment by tylerecouture** [comment]
All XP goes into the same overall "bin", but you can separate XP by tagging quests and badges.  In a student's profile, it will show XP earned for each tag.  So you could tag some items "Math" and some items "English" and see how much XP has been earned by each.

It would look like this:
![image](https://github.com/user-attachments/assets/9cb8fba4-6643-45d5-95e2-61bb63f2e83a)



Alternately (or in addition), you could provide a series of Badges to mark progress in each course separately.  I sort of do that in my own deck, but I created a series of badges per skill instead of per course.  Here's an example of four badges for learning Blender (which is basically the same things as the 3D modelling and Animation Course)
![image](https://github.com/user-attachments/assets/e9094919-4af5-47b5-ab8a-6e2bc1d7b50f)
